tests/integration/backup/one_to_one/import_test.go

```diff
@@ -237,7 +237,8 @@ func TestBackupImport_DoubleRelationshipWithUpdate_NoError(t *testing.T) {
 						},
 					},
 					{
-						"name": "Game of chains",
+						"name":   "Game of chains",
+						"author": nil,
```

Comment:
question: Why has this been added?

Reply:
I believe this was due to the JSON serialization not dropping the nil field here. Potentially it could be an incorrect case within the new `resultsAreEqual` function.

Reply:
It turns out these test case changes are no longer necessary. I've reverted this and a few other changes.

Reply:
Nice, cheers Keenan